applications/nrf5340_audio/src/utils/error_handler.c

```diff
@@ -14,23 +14,37 @@
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(error_handler, CONFIG_ERROR_HANDLER_LOG_LEVEL);
 
-#if (defined(CONFIG_BOARD_NRF5340_AUDIO_DK_NRF5340_CPUAPP) && (CONFIG_DEBUG))
+#if defined(CONFIG_DEBUG)
+struct error_led_setting {
+	struct gpio_dt_spec gpio_spec;
+	int status;
```

Comment:
This may well be correct, but please check that the gpio_pin_configure_dt function takes this argument as an int.

Reply:
Changed status from int to gpio_flags_t.